documentation/smg/12_deploying_releases_to_GitHub.md

```diff
@@ -0,0 +1,89 @@
+# Deploying releases to GitHub
+
+## ci-release-deploy Repository
+
+A repository
+[ci-release-deploy](https://github.com/pace-neutrons/ci-release-deploy)
+has been created to hold the code that implements the deployment pipeline.
+This decision was made so that code is not duplicated between Herbert and Horace
+and to provide fast checkout times for the deploy pipelines.
+
+## Release Pipelines
+
+The release pipelines in ANVIL's Jenkins server should be used to create release
+candidates.
+These pipelines have a `Release-` prefix.
+The pipelines differ from the master builds in that they ensure the
+`-DHORACE_RELEASE_TYPE=release` is passed to CMake.
+It also keeps release candidates separate from the nightly master builds.
+
+## Deploy Pipelines
+
+The deploy pipelines created on ANVIL's Jenkins server should be used to deploy
+release candidates to GitHub.
+Horace and Herbert each have their own deploy pipeline name `Deploy`.
+When triggered, this pipeline takes the following parameters:
+
+- `tag_sha`: The SHA of the Git revision on which to create the release tag
```

Comment:
it would be worth noting whether this is or isn't validated against the build artifacts listed below

Reply:
I'll make this addition when we work out how we're actually doing the validation